connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorUtils.java

```diff
@@ -320,4 +322,19 @@ static void createCompactedTopic(String topicName, short partitions, short repli
     static void createSinglePartitionCompactedTopic(String topicName, short replicationFactor, Admin admin) {
         createCompactedTopic(topicName, (short) 1, replicationFactor, admin);
     }
+
+    static <T> T adminCall(Callable<T> callable, Supplier<String> errMsg)
+            throws ExecutionException, InterruptedException {
+        try {
+            return callable.call();
+        } catch (ExecutionException | InterruptedException e) {
+            if (e.getCause() instanceof TopicAuthorizationException ||
```

Comment:
I think we also need to handle `GroupAuthorizationException` as this can be thrown by `listConsumerGroupOffsets()`

Reply:
Done.